### PR TITLE
i18n - pulling from tx

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

File: locale/fr/foreman_tasks.po [hunk 1]
```diff
@@ -7,28 +7,29 @@
...
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Language: fr\n"
-"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"Plural-Forms: nplurals=3; plural=(n == 0 || n == 1) ? 0 : n != 0 && n % 100000"
+"0 == 0 ? 1 : 2;\n"
 
 msgid "${taskReload ? 'Stop' : 'Start'}  auto-reloading"
-msgstr "${taskReload ? 'Stop' : 'Start'} auto-rechargement"
+msgstr "${taskReload ? 'Stop' : 'Start'}  auto-reloading"
```
Comment (ekohl) [inline locale/fr/foreman_tasks.po:23]: This looks like bad string interpolation from here:
https://github.com/ekohl/foreman-tasks/blob/87f1e943d2675facc8bbdb5c7b0cba9ca28a3e4c/webpack/ForemanTasks/Components/TaskDetails/Components/TaskButtons.js#L51

It's been there for a long time (> 2 years) so consider this a bug report.

**Review by ekohl — COMMENTED** [review]
@adamruzicka these include translations from RH for the next downstream release. It'd be great to have it in a released version.

File: locale/fr/foreman_tasks.po (hunk 1 above)
Comment (ekohl) [reply locale/fr/foreman_tasks.po:23]: I ended up opening https://github.com/theforeman/foreman-tasks/pull/700 while waiting for translations.

**Review by adamruzicka — APPROVED** [review]

**Comment by adamruzicka** [comment]
Thank you @ekohl !

**Pull request merged**